hadoop-common-project/hadoop-common/src/main/java/org/apache/hadoop/fs/statistics/impl/IOStatisticsContext.java

```diff
@@ -31,34 +31,34 @@
  * <p>
  * The {@link #snapshot()} call creates a snapshot of the statistics;
  * <p>
- * The {@link #reset()} call resets the statistics in the current thread so
+ * The {@link #reset()} call resets the statistics in the context so
  * that later snapshots will get the incremental data.
  */
 public interface IOStatisticsContext extends IOStatisticsSource {
```

Comment:
this needs to get moved to org.apache.hadoop.fs.statistics.IOStatisticsContext as it is the public api the apps need; the impl stuff is kept private for the filesystems.